docs/guides/tracks.md

```diff
@@ -1,6 +1,15 @@
 # Tracks
-There are currently three types of tracks
 
-* [AudioTracks](./audio-tracks.md) - allows the selection of alternative AudioTracks for a video
-* [VideoTracks](./video-tracks.md) - allows the selection of an alternative VideoTrack for a video
-* [TextTracks](./text-tracks.md) - Text Tracks are used to display subtitles and captions, and add a menu for navigating between chapters in a video.
+There are currently three types of tracks:
```

Comment:
this would be a good place to put the general track examples (such as addTrack, removeTrack, getTrackById and all the track events)